app/src/main/res/xml/appfilter.xml

```diff
@@ -1197,6 +1208,9 @@
 
     <!-- Material Notes -->
     <item component="ComponentInfo{com.bijoysingh.quicknote/com.bijoysingh.quicknote.activities.MainActivity}" drawable="materialnotes" />
+    
+    <!-- Material Player --> v
+    <item component="ComponentInfo{de.ph1b.audiobook/de.ph1b.audiobook.features.coldStart.SplashActivity}" drawable="materialplayer"/>
```

Comment:
Several of the appfilter entries have a `v` character that will break the XML parsing.